docs/api/top-level-window.md

```diff
@@ -0,0 +1,1240 @@
+# TopLevelWindow
+
+> Create and control native windows.
+
+Process: [Main](../glossary.md#main-process)
```

Comment:
We should probably mention `BrowserWindow` here as the main way to create windows with web content.

Reply:
I'll add detailed information for `TopLevelWindow` and its relationship with `BrowserWindow` in other PRs, so the reviews can be done easier.